src/cosmos/info/thorchainruneInfo.ts

```diff
@@ -15,7 +15,10 @@ const networkInfo: CosmosNetworkInfo = {
   },
   pluginMnemonicKeyName: 'thorchainruneMnemonic',
   shapeshiftApiName: 'thorchain',
-  rpcNode: 'https://rpc.ninerealms.com'
+  rpcNode: {
+    url: 'https://rpc-v1.ninerealms.com',
+    headers: { 'x-client-id': 'ninerealmsClientId' }
```

Comment:
Make this '{{ninerealmsClientId}}' to conform with our EVM RPC apikey replacement policy.